cpp/src/arrow/ipc/read_write_test.cc

```diff
@@ -1242,6 +1245,46 @@ void TestSparseTensorRoundTrip<IndexValueType>::CheckSparseTensorRoundTrip(
   ASSERT_TRUE(result->Equals(sparse_tensor));
 }
 
+template <typename IndexValueType>
+void TestSparseTensorRoundTrip<IndexValueType>::CheckSparseTensorRoundTrip(
```

Comment:
This is almost the same as the CSR version above. Perhaps combine the two using an additional template parameter?

Reply:
Combined them into one.